app/controllers/sessions_controller.rb

```diff
@@ -1,2 +1,30 @@
 class SessionsController < ApplicationController
+
+  def new
+  end
+
+  def create
+    @user = User.find_by(email: params[:session][:email])
+
+    unless @user.nil?
```

Comment:
Actually, maybe not. I'd be tempted to change this block to:

`if @user && @user.authenticate(params[:session][:password])`

I try to avoid nested if-blocks if possible.